### ISSUE TITLE
DisplayLink Manager App Replacing Drivers

### ISSUE THREAD

**Issue by TSPARR** [autopkg/bochoven-recipes#35]
Looks like DisplayLink is deprecating their drivers and moving to using their new app: https://www.displaylink.com/downloads/file?id=1602. The download recipe is picking up this installer correctly, but it's a .pkg, not a .dmg. I haven't dug in far enough to see if it needs repackaging or anything further. Just getting the initial info out there.

**Comment by jc0b** [comment]
To add to this: the release notes for the new app seem to suggest that the drivers will continue to be available (so I agree with @TSPARR's suggestion that the Manager should get its own recipe). 

Unfortunately, it also looks like the download links for the drivers now lead to license agreement pages, and I don't know how easy that would be to bypass.

**Comment by Daz-wallace** [comment]
Looking at the 'current' download recipe, it looks like it's bypassed with some curl options:
https://github.com/autopkg/bochoven-recipes/blob/master/DisplayLink/DisplayLink.download.recipe#L38-L42

Do you have details on where the driver (non-Manager) installers are moving / moved to?

**Comment by TSPARR** [comment]
They seem to still be right there on the same page which is why the recipe is currently grabbing the new app rather than the drivers. Could it be configured to grab the second match? That seems hacky and fragile though...

https://www.displaylink.com/downloads/macos

**Comment by Daz-wallace** [comment]
That'll teach me to read 😉 

I'll take a look and see if we can Reg-Ex for it 👍

**Comment by Daz-wallace** [comment]
I think I have something...

`a href=\"http://assets\.displaylink\.com/live/downloads/release-notes/f(\d+)_DisplayLink\+USB\+Graphics\+Software\+for\+macOS\+.*-Release\+Notes\.txt\" title=\"Release Notes\" class=\"release-note\">Release Notes`

I've got to test it fully but pull request incoming if it works

**Comment by Daz-wallace** [comment]
Nope, no good, it's picking up the Documentation... Back to figuring it out

**Comment by Daz-wallace** [comment]
Ok I'm not sure this is possible without just manually pulling all matches and picking the second one, which is fragile as you mentioned @TSPARR 

I'm guessing we'll need to make a decision on if that'll be ok (or smarter people to take a look 😊 )

**Comment by neilmartin83** [comment]
This regex will get the ID of the first driver that's listed underneath the new app download, but I don't like it - I'll leave it here for folks to comment on/berate as necessary:

`DisplayLink USB Graphics Software for macOS\s.*\s.*\s.*\s.*\s.*\s.*\s.*\s.*\s.*\s.*\s.*a href=\"\/downloads\/file\?id=(\d+)\" title=\"Download File\" onclick=\"\" class=\"download-link\"><button>Download<\/button><\/a>`
